paasta_tools/cli/utils.py

```diff
@@ -438,7 +441,7 @@ def check_ssh_and_sudo_on_master(master, timeout=10):
     with sudo to verify that ssh and sudo work properly. Return a tuple of the
     success status (True or False) and any output from attempting the check.
     """
-    check_command = 'ssh -A -n -o StrictHostKeyChecking=no %s sudo paasta_serviceinit -h' % master
+    check_command = 'ssh -A -n -o StrictHostKeyChecking=no %s sudo /bin/true' % master
```

Comment:
makes sense, but the function name specifically says check_ssh_and_sudo so i assume it's only used in places where sudo is expected?